cmd/nerdctl/start.go

```diff
@@ -41,6 +43,10 @@ func newStartCommand() *cobra.Command {
 		SilenceUsage:      true,
 		SilenceErrors:     true,
 	}
+
+	startCommand.Flags().SetInterspersed(false)
+	startCommand.Flags().BoolP("attach", "a", false, "Attach STDOUT/STDERR and forward signals")
```

Comment:
Could you update the Command Reference in README.md?
Also, please consider adding tests

Reply:
Updated the readme and added tests for both start and start with attach. Hope this looks good.